ml/RL/env.py

```diff
@@ -185,14 +256,124 @@ def reset(self):
                                              heading_angle=self.state[2],
                                              steering_angle=self.state[4]
                                              )
+        self.track = self._create_track()
 
         return self.init_observation
 
     def render(self, mode='human'):
         """
-        Renders an animation to visualize the state of the car.
+        Renders screen window with road represented by line and actual position of car object.
+
+        :return - viewer with drawn objects
         """
-        pass
+
+        x, y, theta, v, psi = self.state
+        center_x, center_y = self.map_center
+        x = x - center_x
+        y = y - center_y
+
+        if self.viewer is None:
+            self.viewer = rendering.Viewer(WINDOW_W, WINDOW_H)
+            left, right, top, bottom = -CAR_WIDTH / 2, CAR_WIDTH / 2, CAR_LENGTH / 2, -CAR_LENGTH / 2
```

Comment:
according to PEP-8 standard for Python max length of line should be 79 characters, this one and some others are longer so pls fix it
NOTE: you can fix this line like this:
left, right, top, bottom = (-CAR_WIDTH / 2, CAR_WIDTH / 2, 
                                      CAR_LENGTH / 2, -CAR_LENGTH / 2)